lit/docs/install/web.lit

```diff
@@ -332,6 +332,47 @@ as performing all pipeline scheduling. It's basically the brain of Concourse.
     this typically means just making your \code{web} node publicly reachable.
   }
 
+  \section{
+    \title{Build log retention}{build-log-retention}
+
+    Build logs are stored in the DB - if they are not cleanup up every once in
+    a while, the storage usage for build logs will continue to grow as more builds
+    run. While this is usually fine for small Concourse instances, as you scale up,
+    you may run into storage concerns.
+
+    To clean up old build logs, Concourse periodically scans for builds whose
```

Comment:
nit: Wondering if it's worth making it clear that you have to configure Concourse to reap builds logs rather than it being automatically on?

```suggestion
    To clean up old build logs, you can configure Concourse to periodically scan for builds whose
```

Reply:
good point, thanks! fixed